autogpt_platform/frontend/src/app/(no-navbar)/layout.tsx

```diff
@@ -8,7 +8,7 @@ export default function OnboardingLayout({
   return (
     <div className="flex min-h-screen w-full items-center justify-center bg-gray-100">
       <div className="mx-auto flex w-full flex-col items-center">
-        {children}
+        <main className="w-full flex-grow">{children}</main>
```

Comment:
we can't we just add flex-frow on the parent div and make it main instead of div ?